### PR TITLE
Homework11: Valeria Kudelko

### PR DESCRIPTION
**Pull request by ValeriaKudelko**

### CODE REVIEW

File: hw11task1.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        """
+        print(f"{self.name} returned '{book.title}'.")
+        book.unreserve()
+
+
+book1 = Book("Mы", "Девид Николс", 292, "978-5-389-08539-8")
+book2 = Book("Я", "Александр Потемкин", 249, "5-902377-10-2")
+user1 = User("Mike")
+user2 = User("Bob")
+user1.borrow_book(book1)
```
Comment (Stml89) [inline hw11task1.py:73]: user1 can take a book, but why user2 can return it?
```suggestion
user1.borrow_book(book1)
user2.return_book(book1)
```

File: hw11task1.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        self.reserved = False
+
+    def is_reserved(self):
+        """
+        In use by another reader.
+        """
+        return self.reserved
+
+
+class User:
```
Comment (Stml89) [inline hw11task1.py:25]: `reserve` functionality is missing

**Review by Stml89 — CHANGES_REQUESTED** [review]

**Comment by Stml89** [comment]
Fix errors in the "Pycodestyle" check

File: .pylintrc [hunk 3]
```diff
@@ -6,9 +6,16 @@ disable=
...
     redefined-builtin,
+    line-too-long,
 
 [REPORTS]
 reports=no
 
 [BASIC]
-good-names=False
\ No newline at end of file
+good-names=False
```
Comment (Stml89) [inline .pylintrc]: are you sure you need to exclude this check?

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: .pylintrc (hunk 3 above)
Comment (ValeriaKudelko) [reply .pylintrc]: deleted

**Review by Stml89 — APPROVED** [review]

**Pull request merged**